src/fidesops/api/v1/scope_registry.py

```diff
@@ -45,6 +45,7 @@
 SAAS_CONFIG_DELETE = "saas_config:delete"
 
 USER_CREATE = "user:create"
+USER_READ = "user:read"
```

Comment:
One subtle change we need to make here is to not only have a scope for reading the current, logged-in, user. We should also have another scope for reading all users in the system. Could we name this scope accordingly? Something like `ALL_USERS_READ` would be clearer, that way we can have another for `CURRENT_USER_READ`. We'd need to follow this same convention then for delete and update too, that can be addressed subsequently.

The alternative here is that we don't have special scopes for users updating their own info, and that on those endpoints we verify that the token making the request has either the user update scope, or belongs to the user being updated.

Reply:
Is it okay that users with the `CURRENT_USER_READ` scope are able to read the data for any user as along as they have the `user_id`? Currently the `get_user` route doesn't validate that it's returning the currently logged in users data.

`CURRENT_USER_READ` makes me thing this scope is limited to reading the currently logged in users data.

Reply:
> Is it okay that users with the CURRENT_USER_READ scope are able to read the data for any user as along as they have the user_id?

`CURRENT_USER_READ`: this should allow the token to read _only_ the logged in user, or nothing at all if the client which issued the token does not have an associated user.

Reply:
Got it. #381 says `GET /api/v1/user/:user_id/: returns just the user requested`. Will the `get_users` function need to have conditional behavior based on the current users scopes?

It could work like this:

- `CURRENT_USER_READ`: Only return the user associated with the current token
- `USER_READ`: Return any user